repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilterChain.java

```diff
@@ -143,9 +147,13 @@ private void splitRequestHeaders(HttpServletRequestWrapper request) {
      */
     private List<FilterContext> getFilterChainForRequest(String uri) {
         List<FilterContext> filters = new LinkedList<>();
-        for (FilterContext filter : filterChainCopy) {
-            if (filter.getUriPattern() == null || filter.getUriPattern().matcher(uri).matches()) {
-                filters.add(filter);
+        if (bypassUrl.isPresent() && Pattern.compile(bypassUrl.get()).matcher(uri).matches()) {
```

Comment:
Does the java optional have a map? i didn't see it when i looked through the list.